docs/user/developing_jobs.md

```diff
@@ -4,17 +4,90 @@ A goal of this plugin is to make it relatively quick and straightforward to deve
 
 Familiarity with [DiffSync](https://diffsync.readthedocs.io/en/latest/) and with developing [Nautobot Jobs](https://nautobot.readthedocs.io/en/latest/additional-features/jobs/) is recommended.
 
-## Quickstart
+## Quickstart Example
 
-As a general overview, here are the different things to be done to have a functioning SSoT job:
+The following code presents a minimum viable example for syncing VLANs from a remote system into Nautobot. You will have to adapt the following things to make this work for your use case:
 
-1. Define the base models that define the data contract
-2. Implement the Nautobot adapter automatically through `nautobot_ssot.contrib.BaseAdapter`
-3. If Nautobot is the source of the job implement `create`, `update`, `delete` methods for the models in the remote system
-4. Implement the remote adapters `load` method, responsible for loading the data from the remote system into the adapter 
-5. Putting it together into a job
+- Swap out any mention of a "remote system" for your actual remote system, such as your IPAM tool
+- Implement any models that you need, unless you are actually only interested in VLAN data
+- Your `load` function in the remote adapter will probably be a little harder to implement, check the example integrations in this repository (under `nautobot_ssot/integrations`)
 
-### Defining the Models
+It contains 3 steps:
+
+- Data modeling
+- Adapters
+    - Nautobot
+    - Remote
+- Nautobot Job
+
+```python
+# example_ssot_plugin/jobs.py
+from typing import Optional
+
+from diffsync import DiffSync
+from nautobot.ipam.models import VLAN
+from nautobot.extras.jobs import Job
+from nautobot_ssot.contrib import NautobotModel, NautobotAdapter
+from nautobot_ssot.jobs import DataSource
+from remote_system import APIClient  # This is only an example
```

Comment:
```suggestion
from . import DictionaryAsAnAPIClient
```

Reply:
The thought is to couple this with a dictionary to be an actual usable example. 

e.g. 

```python
# sot_file.py

DictionaryAsAnAPIClient = {} # actual testable data.
```